packages/system/robotlb/charts/robotlb/templates/role.yaml

```
@@ -0,0 +1,21 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "robotlb.fullname" . }}-cr
+rules:
+ {{- toYaml .Values.serviceAccount.permissions | nindent 2 }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 1

(indentation)